style.css

```diff
@@ -198,6 +198,303 @@ body {
+  padding-left: 8px;
+  padding-top: 10px;
+}
+
+.number__input,
+.minus,
+.plus {
+  font-family: Montserrat, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 0px;
+  cursor: pointer;
+}
+
+.minus {
+  margin-left: 8px;
+  margin-right: 18px;
+}
+
+.number__input {
+  background: transparent;
+  border: none;
+  margin-right: 11px;
+  width: 12px;
+}
+
+.order__cart {
+  background: transparent;
+  border: 1.5px solid #000;
+  display: inline-block;
+  cursor: pointer;
+  font-family: Montserrat, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 13px;
+  margin-bottom: 45px;
+  height: 44px;
+  padding-right: 11px;
+  text-transform: uppercase;
+  width: 253px;
+}
+
+.range-1 {
+  display: inline-block;
+  position: relative;
+  transform: translateX(2px);
+  cursor: pointer;
+}
+
+.range-1__link {
+  color: #000;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 13px;
+  line-height: 16px;
+  margin-right: 10px;
+  text-decoration: none;
+  text-transform: uppercase;
+}
+
+.outfit {
+  background: linear-gradient(to right, #ebe3d8 27%, #000 27% 32%, #ebe3d8 32%);
+  display: flex;
+  padding: 139px 148px 140px 88px;
+  width: 100%;
+}
+
+.outfit__img {
+  display: flex;
+  z-index: 1;
+}
+
+.outfit-details {
+  display: flex;
+  flex-direction: column;
+  margin-top: 116px;
+  z-index: 2;
+}
+
+.outfit-content {
+  background-color: #fae8e4;
+  margin-bottom: 35px;
+  padding: 5px 53px 16px 47px;
+  transform: translateX(-98px);
+  width: 475px;
+}
+
+.outfit__title {
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 41px;
+  margin-bottom: 0;
+}
+
+.outfit__subtitle {
+  font-style: normal;
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 16px;
+  letter-spacing: 0.07em;
+  margin-bottom: 32px;
+  opacity: 0.5;
+  text-transform: uppercase;
+}
+
+.outfit__description {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 21px;
+  margin-bottom: 55px;
+  margin-right: 30px;
+  opacity: 0.5;
+  transform: translateX(3px);
+}
+
+.outfit__cart {
+  height: 44px;
+  width: 253px;
+  background: transparent;
+  border: 1.5px solid #000;
+  font-family: Montserrat, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 13px;
+  line-height: 0px;
+  padding-left: 0;
+  text-transform: uppercase;
+  cursor: pointer;
+}
+
+.range-2 {
+  transform: translateX(33px);
+  cursor: pointer;
+}
+
+.range-2__link {
+  color: #000;
+  cursor: pointer;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 13px;
+  line-height: 16px;
+  margin-right: 10px;
+  text-decoration: none;
+  text-transform: uppercase;
+}
+
+  /*АНИМАЦИЯ ДЛЯ КНОПОК И ССЫЛОК*/
+
+.header__logo:hover,
+.language__img:hover,
+.language__arrow:hover,
+.collection__btn:hover,
+.range:hover,
+.socials-list__item:hover,
+.list__item:hover,
+.order-number:hover,
+.order__cart:hover,
+.range-1:hover,
+.outfit__cart:hover, 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/LOCO_proj_Boiko/pullRequest?prid=5841754)